tests/phpunit/tests/html/wpHtmlTagProcessorBookmarks.php

```diff
@@ -0,0 +1,370 @@
+<?php
+/**
+ * Unit tests covering WP_HTML_Tag_Processor bookmark functionality.
+ *
+ * @package WordPress
+ * @subpackage HTML
+ */
+
+require_once ABSPATH . WPINC . '/wp-html.php';
+
+/**
+ * @group html
+ *
+ * @coversDefaultClass WP_HTML_Tag_Processor
+ */
+class WP_HTML_Tag_Processor_Bookmark_Test extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 56299
+	 *
+	 * @covers set_bookmark
```

Comment:
```suggestion
	 * @covers ::set_bookmark
```
With the `@coversDefaultClass`, the class is correctly omitted from the test method annotation. However, it should still be prefixed with `::`. [Ref](https://phpunit.readthedocs.io/en/9.5/annotations.html#covers)
:1234: Applies elsewhere in the tests.

Reply:
updated in https://github.com/WordPress/wordpress-develop/commit/561acff5157b9db342244f066d7f5017a7e167a2